app-web/gatsby-config.js

```diff
@@ -1,4 +1,7 @@
-require('dotenv').config();
+require('dotenv').config({
```

Comment:
I think dev and test will be the only time we use a `.env` file. The rest of the time it will come from ENV vars. Consider wrapping this in a `if (process.env.NODE_ENV === 'test' || process.env.NODE_ENV === 'development'). In a few projects we have used `nconf` to help manage and merge sensitive and non-sensitive config into a `config` object that can be used like `config.get('sso:clientId')` see https://github.com/bcgov/range-api/tree/develop/server/config

Reply:
I like that approach much better. I'll set up an issue for it to keep track of things. Gatsby has it's own way of passing sensitive/non sensitive data to and from the configuration process to the static site building. So I'll have to see if it's possible to work it in there